### ISSUE TITLE
Add/fork our recipe repo

### ISSUE THREAD

**Issue by choules** [autopkg/autopkg#682]
Dear AutoPkg Team,

we have setup a Github repository with our own recipes: https://github.com/wycomco/wycomco-recipes

It would be great if you could fork this repo to make those recipes available to the AutoPkg community, as described in your Wiki.

Thanks in advance!
Matthias and the [wycomco](https://wycomco.de) Team

**Comment by homebysix** [comment]
Hello @choules -

While reviewing this repo, I had a few questions:

- There are existing ArchiCAD recipes in the MLBZ521-recipes and robperc-recipes repositories. How are the recipes that you are contributing different?

- The ARCHICADUpdate.munki.recipe contains an unnecessary FileFinder step. MunkiImporter automatically knows where to look to import apps located at `%pathname%/*.app`.

- The ARCHICADUpdate.download recipe (and its child recipes) are failing with this error:

    ```
    % autopkg run -v archicad_updates/ARCHICADUpdate.download.recipe
    Processing archicad_updates/ARCHICADUpdate.download.recipe...
    ARCHICADUpdatesProcessor
    ARCHICADUpdatesProcessor: Download URL: https://dl.graphisoft.com/ftp/techsupport/downloads/ac23/ARCHICAD-23-INT-Update-6000-1.0.dmg
    ARCHICADUpdatesProcessor: build: 6000
    ARCHICADUpdatesProcessor: version: 23.0.0.6000
    URLDownloader
    Use of undefined key in variable substitution: 'NAME'
    Command '['/usr/bin/curl', '--silent', '--show-error', '--no-buffer', '--dump-header', '-', '--speed-time', '30', '--location', '--url', 'https://dl.graphisoft.com/ftp/techsupport/downloads/ac23/ARCHICAD-23-INT-Update-6000-1.0.dmg', '--fail', '--output', '~/Library/AutoPkg/Cache/com.github.wycomco.ARCHICADUpdate.download/downloads/tmpvuh905ps']' returned non-zero exit status 35.
    Failed.
    Receipt written to ~/Library/AutoPkg/Cache/com.github.wycomco.ARCHICADUpdate.download/receipts/ARCHICADUpdate.download-receipt-20201108-121053.plist

    The following recipes failed:
        archicad_updates/ARCHICADUpdate.download.recipe
            Error in com.github.wycomco.ARCHICADUpdate.download: Processor: URLDownloader: Error: Command '['/usr/bin/curl', '--silent', '--show-error', '--no-buffer', '--dump-header', '-', '--speed-time', '30', '--location', '--url', 'https://dl.graphisoft.com/ftp/techsupport/downloads/ac23/ARCHICAD-23-INT-Update-6000-1.0.dmg', '--fail', '--output', '~/Library/AutoPkg/Cache/com.github.wycomco.ARCHICADUpdate.download/downloads/tmpvuh905ps']' returned non-zero exit status 35.

    Nothing downloaded, packaged or imported.
    ```

- It's conventional for recipe file names to use the pattern `BIMcollabZOOM.download.recipe`, which is different than the conventional identifier pattern `com.github.wycomco.download.bimcollab_zoom`.

- The BIMcollabZOOM.download recipe is failing with this error. support.bimcollab.com is not loading in my browser, so there may be a problem on their end.

    ```
    % autopkg run -v bimcollab_zoom/com.github.wycomco.bimcollab_zoom.download.recipe
    Processing bimcollab_zoom/com.github.wycomco.bimcollab_zoom.download.recipe...
    URLTextSearcher
    No match found on URL: https://support.bimcollab.com/de/Support/Support/Downloads/BIMcollab-ZOOM
    Failed.
    Receipt written to ~/Library/AutoPkg/Cache/com.github.wycomco.download.bimcollab_zoom/receipts/com.github.wycomco.bimcollab_zoom.download-receipt-20201108-121819.plist

    The following recipes failed:
        bimcollab_zoom/com.github.wycomco.bimcollab_zoom.download.recipe
            Error in com.github.wycomco.download.bimcollab_zoom: Processor: URLTextSearcher: Error: No match found on URL: https://support.bimcollab.com/de/Support/Support/Downloads/BIMcollab-ZOOM

    Nothing downloaded, packaged or imported.
    ```

- Although the "dmg" type recipe is rare, it's usual purpose seems to be creation of a disk image from a zip or other format ([example here](https://github.com/facebook/Recipes-for-AutoPkg/blob/master/SQLDeveloper/SQLDeveloper.dmg.recipe)). Your com.github.wycomco.dmg.sipgate-softphone recipe only does codesign verification and versioning, so I might suggest incorporating those steps into the "download" recipe.

- Your com.github.wycomco.pkg.bimcollab_zoom recipe contains a CodeSignatureVerifier step, which is more commonly found in the "download" recipe.

Could you have a look at those items?

**Comment by jutonium** [comment]
Hello @homebysix
I fixed the bugs and applied your recommendations. The links for BIMcollabZOOM changed indeed.

Regarding ARCHICAD and existing recipes:
The robperc-recipes for the ARCHICAD-Updaters are for versions 17-19 only but the current version is ARCHICAD 24. Ours works for ARCHICAD 20-24.

The MLBZ521-recipes version is for the jss importer and our version is intended for munki.
At the time I wrote our workflow the one provided by MLBZ521-recipes failed while downloading the json containing the download links.
He corrected the errors later and incorporated my changes into the ARCHICADUpdatesProcessor.py script.
Now we could switch to his download-recipe but his workflow still does not contain a munki import step.

The ARCHICAD updates are InstallBuilder-Apps and cannot be drag&dropped to /Applications. The extra FileFinder step is required to get the file name of the app and replace variables in the munki pre- and postinstall_scripts.

Best wishes
Johannes
